src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -1853,20 +1831,30 @@ private function wrapFunctionCallWithReturnValue($returnVariable, $actor, $actio
                 continue;
             }
             if ($args[$i] === "") {
-                $args[$i] = '"' . $args[$i] . '"';
+                $args[$i] = '""';
             }
         }
         if (!is_array($args)) {
             $args = [$args];
         }
         $args = $this->resolveAllRuntimeReferences($args);
         $args = $this->resolveTestVariable($args, $action->getActionOrigin());
-        $output .= implode(", ", array_filter($args, function($value) { return $value !== null; })) . ");";
+        $output .= implode(", ", array_filter($args, $this->filterNullCallback())) . ");";
         return $output;
     }
 
     // @codingStandardsIgnoreEnd
 
+    /**
```

Comment:
Good to add a description of what the function does in header (applies to all new functions added)